helm/tests/testdata/workspace_proxy.yaml

```diff
@@ -0,0 +1,14 @@
+coder:
+  workspaceProxy: true
+
+  image:
+    tag: latest
+
+  env:
+    - name: CODER_PRIMARY_ACCESS_URL
+      value: "https://dev.coder.com"
```

Comment:
Should this be hardcoded?

Reply:
This is a test file, not the actual values.yaml file.